### DISCUSSION TITLE
Power Mode

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1461] [Ideas] [answered]
I would be nice if there was Power Mode in Monkeytype. Power mode was an April Fool's joke Google added to Google Colab. When you type, sparks fly and the screen shakes a little making you feel well, powerful. 
![POWERMODE](https://user-images.githubusercontent.com/79121360/119280500-95800780-bbe6-11eb-9b85-94de38adbc56.gif)

**Comment by ShizukoV** [comment, marked as answer]
Seems cool lol, I would love to see that added :O.

**Reply by unrealapex** [reply]
@Miodec what do you think?

**Reply by Miodec** [reply]
I mean, if you find a library for this or something then sure. Not sure how i would build something like that myself

**Reply by Otard95** [reply]
I'm playing around with it

**Reply by Otard95** [reply]
Could be fun as an easter-egg 😄

**Reply by unrealapex** [reply]
Good idea.

**Reply by Otard95** [reply]
Got the power mode it self setup. Just need the setting and a hidden trigger to unlock it.

**Reply by unrealapex** [reply]
How about clicking the Monkeytype logo on the typing page seven times?

**Reply by Miodec** [reply]
Just make it a hidden command line option like banana caret or monkey.

**Reply by Otard95** [reply]
@UnrealApex Take a look, I hope its what you wanted #1504

**Reply by unrealapex** [reply]
Looks perfect, nice job!

**Reply by ShizukoV** [reply]
Just tried it out, and got a new pb .-.

**Reply by ShizukoV** [reply]
> Just make it a hidden command line option like banana caret or monkey.

wonder what banana caret is 🤔...

**Reply by unrealapex** [reply]
> Just tried it out, and got a new pb .-.

pog

**Reply by ShizukoV** [reply]
> > Just tried it out, and got a new pb .-.
> 
> pog

xD

**Reply by userrand** [reply]
Is it possible to launch it only when the user stays above a certain threshold speed ?

**Reply by unrealapex** [reply]
Not at the moment I believe. It seems like a good suggestion, however, it would probably be a little distracting if it came on with no warning.
